trunner/test_runner.py

```diff
@@ -87,6 +91,37 @@ def dump_logfiles(dut: Dut, dirname: str, logdir: str):
         print(format_logs_for_gh(logs))
 
 
+class CSVCollector:
+    def __init__(self, target):
+        self.filename = f"{target}.csv"
+        self.fieldnames = ["DateStart", "DateEnd", "Catalog", "Type", "Name", "Status", "Run", "Target"]
+        self.data = []
+
+    def add_data(self, test, start, end, status):
+        name = test.name.split("/", 2)
+        entry = {
+            "DateStart": str(start).replace(" ", "T"),
+            "DateEnd": str(end).replace(" ", "T"),
+            "Catalog": name[0],
+            "Type": name[1],
+            "Name": name[2],
+            "Status": re.findall(r"(FAIL|OK|SKIP)", str(status))[0],
```

Comment:
Is this always (FAIL|OK|SKIP)? I don't remember how it's done, but we could consider using `Status` class.